python/HeaderService/hslib_salobj.py

```diff
@@ -468,7 +478,7 @@ def create_dicts(self):
         self.filename_FITS = {}
         self.filename_HDR = {}
 
-    def collect(self, keys):
+    def collect_old(self, keys):
```

Comment:
Since the "collect_old" method is no longer in use, it should be deleted.  If you need to look up something it did later, it can be looked up in older versions of this file via git.

Reply:
Thanks for noticing this. I though I have cleaned up all of them....